data/merchant intro.txt

```diff
@@ -78,17 +78,17 @@ mission "Merchant Start 3"
 	on complete
 		payment 50000
 		conversation
-			`You help the travelers lug his cargo away, still wondering why so much of it is needed. You see the first man walking up to you. "Well, Captain, may I ask you for your name?"`
+			`You help the travelers lug their cargo away, still wondering why so much of it is needed. You see the first merchant you met walking up to you. "Well, Captain, may I ask you for your name?"`
```

Comment:
You could probably drop the "you see" in the second sentence. It's quite unwieldy.
"The first merchant you met walks up to you. 'Well, Captain, ...'"